src/components/ProjectIconSelection/ProjectIconSelection.stories.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+import React, { Fragment } from 'react';
+import { storiesOf } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info';
+import { action } from '@storybook/addon-actions';
+
+import Showcase from '../../../.storybook/components/Showcase';
+import ProjectIconSelection from './ProjectIconSelection';
+
+storiesOf('ProjectIconSelection', module).add(
+  'default',
+  withInfo()(() => (
+    <Fragment>
+      <Showcase label="Image selected">
+        <ProjectIconSelection
+          selectedIcon={'/static/media/icon_pineapple.5d31b188.jpg'}
```

Comment:
Nit: The squiggly brackets aren't necessary here. `selectedIcon="/static/media/icon_pineapple.5d31b188.jpg"`

(When I say "Nit: ", it means that it's a nitpick, and an optional thing to fix. It doesn't really matter :) you can quickly tweak it if you have the time, but don't stress about it if you're busy)